third_party/move/move-prover/bytecode-pipeline/src/eliminate_imm_refs.rs

```diff
@@ -116,7 +116,7 @@ impl<'a> EliminateImmRefs<'a> {
                         aa,
                     ));
                 },
-                Destroy if self.is_imm_ref(srcs[0]) => {
+                Drop if self.is_imm_ref(srcs[0]) => {
                     // skip the destroy on an immutable ref
```

Comment:
maybe replace "destroy" with "skip" in the comment as well?

Reply:
Done